### ISSUE TITLE
Label of radios rendered underneath the elements in Version 4.1.1

### ISSUE THREAD

**Issue by B-Kluss** [neilime/twbs-helper-module#324]
Same as in issue #294 but for version 4.1.1

**Describe the bug**
The label for a radio-list is rendered underneath the elements instead of above.

```
echo $this->formRow(
    $factory->create(
        [
            'name'       => 'flexRadioDefault',
            'type'       => 'radio',
            'options'    => [
               'label' => Choose from the list below
                'form_group'    => false,
                'value_options' => [
                    [
                        'label'      => 'Default radio',
                        'value'      => 'option1',
                        'attributes' => ['id' => 'flexRadioDefault1'],
                    ],
                    [
                        'label'      => 'Default checked radio',
                        'value'      => 'option2',
                        'attributes' => ['id' => 'flexRadioDefault2'],
                    ],
                ],
            ],
            'attributes' => [    'value' => 'option2'    ],
        ]
    )
);
```

**Actual behavior**

o Default radio
o Default checked radio
Choose from the list below

**Expected behavior**

Choose from the list below
o Default radio
o Default checked radio

**Comment by driehle** [comment]
@neilime I can confirm this issue, would it be possible to backport the fix from #294 into the 4.x series?

**Comment by neilime** [comment]
Hi @driehle thank you for offering your help, I've create the dedicated branch: https://github.com/neilime/twbs-helper-module/tree/4.x
